wasm/example/c++_get_started/main.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:io';
+import 'package:wasm/wasm.dart';
+
+/// Start by installing the tools needed to build the Wasmer runtime:
```

Comment:
I think it'd make more sense to put all this in a README.md

Reply:
Why?

Reply:
Readmes are where documentation about the package usually goes. Code comments are good for API documentation, but readmes are where people go when they want to know about things like what the package does, and how to set it up.

Reply:
I've restored the package-level README.